packages/console/src/ds-components/Uploader/FileUploader/index.tsx

```diff
@@ -90,10 +88,10 @@ function FileUploader({
         return;
       }
 
-      const fileSizeLimit = Math.min(maxSize, maxUploadFileSize);
-
-      if (acceptedFile.size > fileSizeLimit) {
-        setUploadError(t('components.uploader.error_file_size', { size: fileSizeLimit / 1024 }));
+      if (acceptedFile.size > maxUploadImageSize) {
```

Comment:
Should not use `maxUploadImageSize` here since this is a general `<FileUploader />`?